src/main/java/com/simibubi/create/compat/jei/category/CreateRecipeCategory.java

```diff
@@ -124,7 +124,7 @@ public static IRecipeSlotRichTooltipCallback addStochasticTooltip(ProcessingOutp
 	@SuppressWarnings("removal") // see below
 	public static IRecipeSlotBuilder addFluidSlot(IRecipeLayoutBuilder builder, int x, int y, FluidIngredient ingredient) {
 		int amount = ingredient.getRequiredAmount();
-		return builder.addSlot(RecipeIngredientRole.OUTPUT, x, y)
+		return builder.addSlot(RecipeIngredientRole.INPUT, x, y)
```

Comment:
This implementation is not correct, not all fluid slot in Create's recipe category are input, you should add a overload accepting the role as parameter, deprecating the current one and mark for removal, and then replace all calls with correct role.